client/gulpfile.js

```diff
@@ -90,35 +91,48 @@ gulp.task('sass:watch', function () {
     gulp.watch('assets/styles/sass/*.scss', ['compile-sass']);
 });
 
-gulp.task('create-dev-config', function (){
+gulp.task('copy_images_to_dist', function () {
+    /* Copy the images in the image folder to a dist folder */
+    return gulp.src(paths.images)
+        .pipe(gulp.dest('../server/web/static/dist/assets/img'));
+});
+
+gulp.task('create-dev-config', function () {
     /** Creates a config file for Angular with the relevant environment variables for development */
-   return gulp.src('taskingmanager.config.json')
-       .pipe(config('taskingmanager.config', {environment: 'development'}))
-       .pipe(gulp.dest('app'))
+    return gulp.src('taskingmanager.config.json')
+        .pipe(config('taskingmanager.config', {environment: 'development'}))
+        .pipe(gulp.dest('app'))
 });
 
-gulp.task('create-release-config', function (){
+gulp.task('create-release-config', function () {
     /** Creates a config file for Angular with the relevant environment variables for release */
-   return gulp.src('taskingmanager.config.json')
-       .pipe(config('taskingmanager.config', {environment: 'release'}))
-       .pipe(gulp.dest('app'))
+    return gulp.src('taskingmanager.config.json')
+        .pipe(config('taskingmanager.config', {environment: 'release'}))
+        .pipe(gulp.dest('app'))
 });
 
 /** Build task for will minify the app and copy it to the dist folder ready to deploy */
-gulp.task('build', function(callback) {
+gulp.task('build', function (callback) {
     runSequence('clean',
-                'create-release-config',
-                'compile-sass',
-                'minify-css',
-                'uglify',
-                'processhtml',
-                callback);
+        'create-release-config',
+        'compile-sass',
+        'copy_images_to_dist',
+        'minify-css',
+        'uglify',
+        'processhtml',
+        callback
+    )
+    ;
 });
 
-gulp.task('run', function(callback){
+gulp.task('run', function (callback) {
     runSequence('create-dev-config',
-                'compile-sass',
-                'browser-sync',
-                'sass:watch',
-                callback);
-});
\ No newline at end of file
+        'compile-sass',
+        'copy_images_to_dist',
```

Comment:
we don't need to copy images to dist when running the task, only when building it.